src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1892,12 +1892,33 @@ static bool processSubscriptions
       //
       if (subCacheActive == false)
       {
-        BSONObj query  = BSON("_id" << OID(mapSubId));
-        BSONObj update = BSON("$set" <<
-                              BSON(CSUB_LASTNOTIFICATION << rightNow) <<
-                              "$inc" << BSON(CSUB_COUNT << (long long) 1));
+        BSONObj subOrig;
+        std::string err;
```

Comment:
This `err` definition collides with existing `err` (which comes from L1769). Thus , I'd suggest to define a new `newErr` or something like that for this.

In addition, collectionUpdate() in L1921 should be as in the original code and use `err` and not the newly defined string.

Reply:
fixed in b2b21a1